src/python/pants/engine/internals/native.py

```diff
@@ -71,18 +71,18 @@ def generator_send(
             if isinstance(res, Get):
                 # Get.
                 return PyGeneratorResponseGet(
-                    res.output_type,
-                    res.input_type,
-                    res.input,
+                    product=res.output_type,
+                    declared_subject=res.input_type,
+                    subject=res.input,
                 )
             elif type(res) in (tuple, list):
                 # GetMulti.
                 return PyGeneratorResponseGetMulti(
-                    tuple(
+                    gets=tuple(
                         PyGeneratorResponseGet(
-                            get.output_type,
-                            get.input_type,
-                            get.input,
+                            product=get.output_type,
+                            declared_subject=get.input_type,
+                            subject=get.input,
```

Comment:
Hm, fine to not fix, but I thought I had fixed most of the Rust code to call this `input` and `output`.